core_lib_generator/template_core_lib/core_lib/template_core_lib.py

```diff
@@ -0,0 +1,19 @@
+from omegaconf import DictConfig
+from core_lib.core_lib import CoreLib
+
+# template_cache_handler_imports
+# template_data_handlers_imports
+# template_da_imports
+
+
+class TemplateCoreLib(CoreLib):
```

Comment:
TemplateCoreLib is corrent.
in other places, it's only the `Template`
here it's good change in other places